Dockerfile

```diff
@@ -27,10 +28,10 @@ RUN cp -f /usr/share/zoneinfo/America/Los_Angeles /etc/localtime && \
 # apk add autoconf aclocal automake libtool
 # tar -xvzpf libffi-3.2.1.tar.gz
 # ./configure --prefix=/usr/local
-RUN mkdir -p /tmp/ffi && \
-  curl -sL https://codeload.github.com/libffi/libffi/tar.gz/refs/tags/v3.2.1 \
-  | tar -xz -C /tmp/ffi && cd /tmp/ffi/libffi-3.2.1 && ./autogen.sh &&\
-  ./configure --prefix=/usr/local && make && make install && rm -rf /tmp/ffi
+#RUN mkdir -p /tmp/ffi && \
```

Comment:
as long as `apk add libffi` > 3.2.1 is working, let's get rid of this whole comment block